sections/discussion/ourprocessreflections.tex

```diff
@@ -0,0 +1,58 @@
+\section{Reflections on our process}
+In \autoref{sec:our-process} we described our process and the things that we used from scrum.
+In this section we reflect on our process, the changes that we had done throughout the semester as well as what we would change for our future projects.
+
+\subsubsection{Scrum}
+Our scrum inspired process worked well with sprints, sprint planning, sprint backlog, product owner, product backlog and product increment.
+We took turns on having the role as PO.
+The PO had the task of planning the next product backlog and add a DoD onto every task in the product backlog.
+The sprint backlog and product backlog can be seen on \autoref{fig:trello-kanban}. 
+The leftmost column is the product backlog, the next four column are the sprint backlog.
+These are split into \textit{Sprint Backlog}, \textit{In progress}, \textit{In review} and \textit{Done}.
+This gives us a better overview on how far people are with their tasks.
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.8\linewidth]{figures/trellopicture.PNG}
+    \caption{Our kanban board for the product and sprint backlog on Trello}
+    \label{fig:trello-kanban}
+\end{figure}
+\noindent
+When we conducted the sprint planning, we would discuss the every element in the product backlog and estimate how many story points each task should have.
```

Comment:
det er vel kun i den proposed sprint backlog? ellers skulle vi gøre det for alle opgaver der findes i vores trello